dune-project

```diff
@@ -113,27 +113,27 @@ should also be fine under Windows too.
   logs
   (ounit :with-test)))
 
-(package
- (name cohttp-lwt-jsoo)
- (synopsis "CoHTTP implementation for the Js_of_ocaml JavaScript compiler")
- (description "\
-An implementation of an HTTP client for JavaScript, but using the
-CoHTTP types.  This lets you build HTTP clients that can compile
-natively (using one of the other Cohttp backends such as `cohttp-lwt-unix`)
-and also to native JavaScript via js_of_ocaml.
-")
- (depends
-  (ocaml (>= 4.08))
-  (http (= :version))
-  (cohttp (= :version))
-  (cohttp-lwt (= :version))
-  logs
-  (lwt (>= 3.0.0))
-  (lwt_ppx :with-test)
-  (conf-npm :with-test)
-  (js_of_ocaml (>= 3.3.0))
-  (js_of_ocaml-ppx (>= 3.3.0))
-  (js_of_ocaml-lwt (>= 3.5.0))))
+; (package
+;  (name cohttp-lwt-jsoo)
+;  (synopsis "CoHTTP implementation for the Js_of_ocaml JavaScript compiler")
+;  (description "\
+; An implementation of an HTTP client for JavaScript, but using the
+; CoHTTP types.  This lets you build HTTP clients that can compile
+; natively (using one of the other Cohttp backends such as `cohttp-lwt-unix`)
+; and also to native JavaScript via js_of_ocaml.
+; ")
+;  (depends
+;   (ocaml (>= 4.08))
+;   (http (= :version))
+;   (cohttp (= :version))
+;   (cohttp-lwt (= :version))
+;   logs
+;   (lwt (>= 3.0.0))
+;   (lwt_ppx :with-test)
+;   (conf-npm :with-test)
+;   (js_of_ocaml (>= 3.3.0))
+;   (js_of_ocaml-ppx (>= 3.3.0))
+;   (js_of_ocaml-lwt (>= 3.5.0))))
```

Comment:
Why is this commented out?

Reply:
jsoo doesn't build on 4.12+domains switch.